docs/pages/04.02-callbacks.md

```diff
@@ -114,6 +114,23 @@ buttonIconHandler(data, callbackIcon) {
 }
 ```
 
+### propertyIconHandler()
+Called when a user wants to pass in a specific object to a dropdown menu. The propertyIconHandler expects a Carbon jsx object as the return value from the callback. This is used to display the Carbon icon in the dropdown menu. The propertyIconHandler provides the following data:
```

Comment:
` Carbon jsx object`
Should just be `jsx object`

`This is used to display the Carbon icon in the dropdown menu`
Should be changed to 
`This is used to display the jxx object in the dropdown menu`